src/Graph/GraphPrinter.php

```diff
@@ -298,7 +134,7 @@ protected function processResultRow( array /* of SMWResultArray */ $row ) {
 				// create SRF\GraphNode for column 0
 				if ( $i == 0 ) {
 					$node = new GraphNode( $object->getShortWikiText() );
-					$node->setLabel($object->getDisplayTitle());
+					$node->setLabel( $object->getDisplayTitle() );
```

Comment:
@RacoonDev `getDisplayTitle` is not guaranteed to have a string representation except for when a display title is available (as the name suggests), meaning that if a page has no display title (aka DISPLAYTILE) then this function will return with an empty string therefore you should rely on `getPreferredCaption` instead to have some title representation.

Reply:
@mwjames thanks for the advice, I've replaced the functions. I think there is no reason to overwrite the "getPreferredCaption" but please tell me if I'm wrong.